webui/tests/cypress/integration/02-tasks.spec.ts

```diff
@@ -17,11 +17,10 @@ describe('Task List', () => {
 
     it('should terminate notebook', () => {
       cy.get('button[aria-label="Notebook"]').click();
-      cy.server();
-      cy.route('DELETE', /\/notebook.*/).as('terminateRequest');
       cy.get(`${recordSelector}:first-child .ant-dropdown-trigger`).click();
       cy.get('.ant-dropdown .ant-dropdown-menu-item').contains(/kill/i).click();
-      cy.wait('@terminateRequest');
+      // Using the server/route approach to detect endpoint calls does not work with new API
```

Comment:
@hkang1 did it not work with POST method either? this changed after kill task operations migrated to new API. 

I remember reading somewhere that Cypress couldn't intercept native fetch requests :thinking: 

It looks like in the new version they're changing this https://docs.cypress.io/api/commands/server.html#Syntax